mangaki/mangaki/templates/fix/fix_index.html

```diff
@@ -0,0 +1,52 @@
+{% extends "base.html" %}
+{% load staticfiles %}
+
+{% block title %}Liste des suggestions {% if suggestions.paginator.page_range|length > 1 %}- Page {{ suggestions.number }} {% endif %}- Mangaki{% endblock %}
+
+{% block body %}
+
+<h1>Liste des suggestions</h1><br/>
+
+{% for suggestion in suggestions %}
+<div class="row">
+<div class="panel panel-default">
+    <h3 class="panel-heading panel-title">
+        <a href="{% url "fix-suggestion" suggestion.id %}" class="text-primary">Suggestion #{{ suggestion.id }}</a>
+        <span class="small font-weight-bold">
+            &nbsp;&nbsp;<span class="glyphicon glyphicon-{% if suggestion.is_checked %}lock{% else %}flag{% endif %}" aria-hidden="true"></span>
+            Ce problème est {% if suggestion.is_checked %}fermé{% else %}en cours{% endif %}
```

Comment:
I think the terminology behind `is_checked` is more like:
- an admin has checked a suggestion and valid its contents and triggered actions (or not)
So rather than "fermé", "résolu" seems better?